client/src/AppBundle/Service/Client/RestClient.php

```diff
@@ -255,6 +255,8 @@ public function post($endpoint, $mixed, array $jmsGroups = [], $expectedResponse
             $options['deserialise_groups'] = $jmsGroups;
         }
 
+        file_put_contents('php://stderr', print_r("ENDPOINT IS: \n\n", TRUE));
+        file_put_contents('php://stderr', print_r($mixed, TRUE));
```

Comment:
Do we still want these?

Reply:
Aaaargh. Nooo.. Next story I do I'm gonna hit you up for getting the proper debugging set up.